code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -300,3 +310,83 @@
 			to_chat(user, "Nearest operative detected is <i>[nearest_op.real_name].</i>")
 		else
 			to_chat(user, "No operatives detected within scanning range.")
+
+/obj/item/weapon/pinpointer/crew
+	name = "crew pinpointer"
+	desc = "A handheld tracking device that points to crew suit sensors."
+	shows_nuke_timer = FALSE
+	icon_state = "pinoff_crew"
+	icon_off = "pinoff_crew"
+	icon_null = "pinonnull_crew"
+	icon_direct = "pinondirect_crew"
+	icon_close = "pinonclose_crew"
+	icon_medium = "pinonmedium_crew"
+	icon_far = "pinonfar_crew"
+
+/obj/item/weapon/pinpointer/crew/proc/trackable(mob/living/carbon/human/H)
+	var/turf/here = get_turf(src)
+	if((H.z == 0 || H.z == here.z) && istype(H.w_uniform, /obj/item/clothing/under))
+		var/obj/item/clothing/under/U = H.w_uniform
+
+		// Suit sensors must be on maximum.
+		if(!U.has_sensor || U.sensor_mode < 3)
+			return FALSE
+
+		var/turf/there = get_turf(H)
+		return (H.z != 0 || (there && there.z == H.z))
+
+	return FALSE
+
+/obj/item/weapon/pinpointer/crew/attack_self(mob/living/user)
+	if(active)
+		active = FALSE
+		icon_state = icon_off
+		user.visible_message("<span class='notice'>[user] deactivates their pinpointer.</span>", "<span class='notice'>You deactivate your pinpointer.</span>")
+		return
+
+	var/list/name_counts = list()
+	var/list/names = list()
+
+	for(var/mob/living/carbon/human/H in mob_list)

```

Comment:
Is this going to respect Z-Levels, or are you going to get it pointing at the corresponding place in Wild West, when they're on the station?

Reply:
In the trackable proc, it checks that they're on the same z-level. Is Wild West a gateway level?

Reply:
Thats good.

(And yes)